backend/core/.env.template

```diff
@@ -1,6 +1,7 @@
 PORT=3100
 NODE_ENV=development
 DATABASE_URL=postgres://bloom-dev:CHANGE-ME@localhost:5432/bloom
+REDIS_URL=redis://127.0.0.1:6379/0
```

Comment:
I just noticed that Heroku added two URLs to the container config for Redis and one of them is SSL, so I think the connection to redis will be over a public internet and we should use SSL in that case.

Reply:
@pbn4 redis over TLS applied